### ISSUE TITLE
[a11y] Cannot log out

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1212]
**Describe the bug**
Logging out does not work.

![image](https://user-images.githubusercontent.com/37256050/161287086-95f9e0d5-5d21-41eb-b88c-12df21e9e646.png)

Edge, macOS
